docker/release/Dockerfile

```diff
@@ -11,5 +11,6 @@ RUN apt-get install -y /tmp/iroha.deb; \
 WORKDIR /opt/iroha_data
 
 COPY entrypoint.sh /entrypoint.sh
+COPY wait-for-it.sh /wait-for-it.sh
```

Comment:
Use `COPY entrypoint.sh wait-for-it.sh /` instead, so there's no additional layer

Also checkout [this link](https://stackoverflow.com/questions/30256386/how-to-copy-multiple-files-in-one-layer-using-a-dockerfile), I might miss smth